website_sale_checkout_skip_payment/__manifest__.py

```diff
@@ -20,7 +20,7 @@
         "views/res_config_settings_views.xml",
     ],
     "assets": {
-        "web.assets_frontend": [
+        "web.assets_tests": [
             "website_sale_checkout_skip_payment/static/src/js/*.js",
```

Comment:
Shouldn't this be reorganized for not importing as glob the general JS folder (and avoiding side effects if something more is added there)? Maybe `website_sale_checkout_skip_payment/static/src/js/tours/*.js`

Reply:
Done in src/tests/tours in Odoo's style